apps/ycrv/components/list/GaugeListRow.tsx

```diff
@@ -26,12 +26,13 @@ type TGaugeListRow = {
 }
 
 function GaugeListRow({gauge, gaugeVotes, votesState, votesDispatch}: TGaugeListRow): ReactElement | null {
-	const {vote} = useVLyCRV();
+	const {vote, mutateData} = useVLyCRV();
 	const {toast} = yToast();
 	const {provider, isActive} = useWeb3();
 	const [currentVotes, set_currentVotes] = useState<string>('');
+	const [txStatusVote, set_txStatusVote] = useState(defaultTxStatus);
 
-	useMemo((): void => {
+	useEffect((): void => {
```

Comment:
Are we sure this is needed? `useEffect` should be mostly used for external requests